### PR TITLE
1 add landing message

### PR DESCRIPTION
**Pull request by BlackSecretToken**
Add first landing page.

### CODE REVIEW

**Comment by BlackSecretToken** [comment]
Could you remove enjoy letters and insert hello instead

**Comment by HighVisionn** [comment]
Could you remove enjoy letters and insert hello instead.

File: index.html [hunk 1]
```diff
@@ -5,5 +5,6 @@
     </head>
     <body>
         <h1>Welcome to Potionfy! Enjoy your stay</h1>
+        <h2>Welcome to Potionfy! Enjoy your stay</h2>
```
Comment (HighVisionn) [inline index.html:8]: Please remove the Enjoy letter and instead add Happy

Comment (HighVisionn) [reply index.html:8]: Okay. I will change.

Comment (BlackSecretToken) [reply index.html:8]: Okay. I will change.

**Review by HighVisionn — CHANGES_REQUESTED** [review]
Please change code according to my comments.

**Review by HighVisionn — APPROVED** [review]
Well Done!

**Pull request merged**